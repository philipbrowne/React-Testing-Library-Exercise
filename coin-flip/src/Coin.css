.Coin-Img {
  width: 200px;
  margin-top: 1rem;
}
