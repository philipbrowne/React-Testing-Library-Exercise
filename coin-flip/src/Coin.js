import React, { useState } from 'react';
import './Coin.css';

const Coin = ({ img }) => {
  return (
    <div className="Coin">
      <img className="Coin-Img" src={img} alt="coin" />
    </div>
  );
};

export default Coin;
