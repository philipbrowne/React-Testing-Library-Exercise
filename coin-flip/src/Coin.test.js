import React from 'react';
import { render, screen } from '@testing-library/react';
import Coin from './Coin';

it('renders without crashing', () => {
  render(<Coin />);
});

it('should match snapshot', () => {
  const { asFragment } = render(<Coin />);
  expect(asFragment()).toMatchSnapshot();
});
