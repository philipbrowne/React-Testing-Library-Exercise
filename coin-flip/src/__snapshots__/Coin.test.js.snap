// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<DocumentFragment>
  <div
    class="Coin"
  >
    <img
      alt="coin"
      class="Coin-Img"
    />
  </div>
</DocumentFragment>
`;
